server.py

```diff
@@ -0,0 +1,65 @@
+import json
+import socket
+
+class Server(object):
```

Comment:
We talked about this before, but just want to write it down somewhere.
1. The Car RPi socket is initially acting as a server while in AP mode where it waits for requests and serves responses, which is what is implemented here.
2. Once connected to Wi-Fi, the Car RPi socket acts as a client where it will send either a "register_car" request or a "connect_car" request and wait for a response message from the Server RPi.
3. Once connected to the server, the Car RPi socket is back to acting as a server where it just continuously processes "movement_control" messages coming from the Server RPi.

So, the Car RPi code will eventually need to handle acting as a client and a server. Maybe we can use multiple sockets for this, but it should be doable with just one.